docs/en/CSMM/allocs.md

```diff
@@ -9,10 +9,10 @@ If you are hosting your game with a professional hosting company, your game serv
 The first step is to confirm whether Alloc's Fixes is installed and running properly. To do that, you can simply type 'version' into your console or RCON window. The output should look something like below. There are three separate Alloc's Fixes mods that need to be installed, and depending on the version you install, the numbers (18, 21, 31) can be different.
```

Comment:
Could you change `the numbers (18, 21, 31) can be different.` to `the numbers (20, 23, 35) can be different.`
as well please?